docs/SUMMARY.md

```diff
@@ -31,6 +32,9 @@
   - [Components](./dev-guide/components.md)
   - [Operations](./dev-guide/dev-operations.md)
   - [Sui structures](./dev-guide/sui-struct.md)
+- [Operator Guide](./operator-guide/operator-guide.md)
+  - [Storage Node](./operator-guide/storage-node.md)
+  - [Aggregator](./operator-guide/aggregator.md)
```

Comment:
Suggestion (for a follow-up): Move both the developer and operator guide *after* the general setup and usage instructions.

cc @gdanezis